pom.xml

```diff
@@ -233,7 +233,7 @@
                 </property>
             </activation>
             <properties>
-                <build.number>Milestone Build</build.number>
+                <build.number>Milestone Build 6</build.number>
```

Comment:
Seems like the `build.number` content is also used when logging into the console. So that would cause the M6 part to be duplicated:

```
                          __  _____    ____      
  ____  ____  ___  ____  / / / /   |  / __ )     
 / __ \/ __ \/ _ \/ __ \/ /_/ / /| | / __  | 
/ /_/ / /_/ /  __/ / / / __  / ___ |/ /_/ /      
\____/ .___/\___/_/ /_/_/ /_/_/  |_/_____/     
    /_/                        2.4.0.M6
                               Milestone Build
```